### DISCUSSION TITLE
wildcard for fallback endpoint is not working in @get()

### DISCUSSION THREAD

**Discussion by chennavamshikrishna** [micronaut-projects/micronaut-core#6947] [Q&A]
### Expected Behavior

I created a get endpoint with * as value in a controller.I want to this endpoint to called as a fallback if none of endpoint in a controller matches

### Actual Behaviour

when I tried to endpoint  with value not present in controller fallback endpoint is not getting triggered instead i am getting 404 

### Steps To Reproduce

@Get(value=“*”)
public string getValue(){
return “apple”;
}

### Environment Information

ubuntu java version 11

### Version

3.3

**Comment by graemerocher** [comment]
`*` is not a valid uri template https://datatracker.ietf.org/doc/html/rfc6570

**Reply by chennavamshikrishna** [reply]
@graemerocher 
Is there any work around for creating fall back endpoint in micronaut

In spring we can achieve by using *.
Can please suggest

**Reply by chennavamshikrishna** [reply]
even /** is not working in @get()

**Comment by tibistibi** [comment]
+1

`@Controller("/api/mock")
public class MockController {
    private static final Logger LOG = LoggerFactory.getLogger(MockController.class);

    @Get("/{**anyPath}")
    void mock2(@PathVariable String anyPath) {
        LOG.trace("mock do nothing, {}", anyPath);
    }

}`

and that would match 
/api/mock/abc
/api/mock/abc/123
/api/mock/abc/123/def
etc...
